crates/steel-core/src/steel_vm/primitives.rs

```diff
@@ -790,11 +820,12 @@ fn identity_module() -> BuiltInModule {
     module
         // .register_value("int?", gen_pred!(IntV))
         .register_native_fn_definition(NOT_DEFINITION)
-        .register_native_fn_definition(INTEGERP_DEFINITION)
+        .register_native_fn_definition(NUMBERP_DEFINITION)
         .register_native_fn_definition(INTP_DEFINITION)
+        .register_native_fn_definition(INTEGERP_DEFINITION)
         .register_native_fn_definition(FLOATP_DEFINITION)
-        .register_native_fn_definition(NUMBERP_DEFINITION)
```

Comment:
Is there a way to (unit) test that a symbol isn't registered more than once?

Reply:
Could do something within the `register_native_fn_definition` or `register_value` etc functions that is something like:

```rust
#[cfg(debug_assertions)]
assert!(!module.contains(symbol));
```

Just hand waving a bit, but basically in debug mode assert that the symbol isn't already registered in the module

Reply:
Ack.